websites/T/Tanks.gg/presence.ts

```diff
@@ -0,0 +1,105 @@
+const presence = new Presence({
+		clientId: "1003195428642238464",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/PjDWjpk.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = window.location;
+
+	if (pathname === "/") presenceData.details = "Browsing the home page";
+	else if (document.location.pathname.startsWith("/tank/")) {
+		presenceData.details = "Viewing a tank";
+		presenceData.state = `${document
+			.querySelector("head > title")
+			.textContent.replace(" - World of Tanks - tanks.gg", "")}`;
```

Comment:
again why???

Reply:
I tried to use the selector that included the text I wanted to display, but this error would always come back if you attempted to view a different country, which I couldn't fix unfortunately 
![image](https://user-images.githubusercontent.com/65370077/182045242-5664524f-1792-4739-9896-10a222cb1946.png)

Reply:
Idk what selector you used, but with this
`document.querySelector(".header-tank > h1").childNodes[0].textContent` you can have the name of the tank, and this part of the page don't change with the country I think.
Tested on https://tanks.gg/tank/m-v-y and this work.
Btw, just suggestion, but if this selector works, then you can add if the tank is versatile, etc, and much more, this could be interesting to add more features to your presences.

Reply:
Oops my bad, I was looking at the tech tree page and code. I'll implement this selector for the tanks, thanks

Reply:
@DarkVillager @RisingSunLight42 Thanks to both of you for bringing this to my attention, by the way, it helped me fix an issue that I couldn't figure out before